test/mocks/MockMicroserviceAuthConnector.scala

```diff
@@ -28,7 +29,7 @@ trait MockMicroserviceAuthConnector extends TestSupport with BeforeAndAfterEach
 
   val mockMicroserviceAuthConnector: MicroserviceAuthConnector = mock[MicroserviceAuthConnector]
 
-  def mockAuth(response: Future[Unit] = Future.successful(())): Future[Nothing] = {
+  def mockAuth(response: Future[Any] = Future.successful(ConfidenceLevel.L250)): Future[Nothing] = {
```

Comment:
I think this could be changed to Future[ConfidenceLevel] as it shouldn't be taking in any other type?